### DISCUSSION TITLE
Working with Emilia - Easy Question. How do I change the main header on home page?

### DISCUSSION THREAD

**Discussion by clarkedarkmatter** [LekoArts/gatsby-themes#759] [Questions]
I'm still trying to learn shadowing. Seems like a lot of others are having similar issues with what should be easy tasks. 

Can someone break down a play-by-play/ELI5 to making some easy edits to title/meta/etc.? I feel hopeless right now on this theme.

**Comment by ch4nd4n** [comment]
I believe you can update that in `gatsby-config.js`
For example my site it's something like

Source: https://github.com/ch4nd4n/chandankumar.com/blob/master/gatsby-config.js

```javascript
siteMetadata: {
    author: "@ch4nd4n",
    siteTitle: `Chandan Kumar`,
    siteHeadline: `Chandan Kumar`,
    siteUrl: `https://www.chandankumar.com`,
    siteDescription:
      "blah blah",
    siteTitleAlt: `Chandan Kumar`,
  },
```

**Comment by clarkedarkmatter** [comment]
I tried that but it didn't work. Thank you for trying though.

**Reply by ch4nd4n** [reply]
Just iterating but you don't need theme shadowing for Meta tags. Can you tell me how are you verifying meta tags and what values are you seeing for it?

**Comment by flamypow** [comment]
If you're talking about changing the text in the header, you have to change values under options for the starter. 
https://github.com/LekoArts/gatsby-themes/blob/968bbb4c6ebc7ac728e2371a7d5e8a286a2713ab/themes/gatsby-theme-emilia/README.md
